settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
Oh ok, now I understand the reason behind this 👍 . Can you please update your PR #50 with some readme changes to make this clear? I'll be happy to merge it in here before merging this.